lib/src/modules/instabug.dart

```diff
@@ -153,12 +153,25 @@ class Instabug {
   /// it on your dashboard.
   /// The [invocationEvents] are the events that invoke
   /// the SDK's UI.
+  static Future<void> init({
```

Comment:
question: Should we add an optional parameter for `debugLogsLevel` and do the associated changes in this PR as well?

Reply:
This will be handled in a separate PR